src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -129,6 +129,37 @@ public static int parseInsurancePlan(String insurancePlanId) throws ParseExcepti
         return trimmedInsurancePlanId;
     }
 
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given claimId is invalid based on preset conventions.
+     */
+    public static String parseClaimId(String claimId) throws ParseException {
+        requireNonNull(claimId);
+        return claimId.trim();
+    }
+
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the claim amount is not a valid positive integer
+     */
+    public static int parseClaimAmount(String claimAmount) throws ParseException {
+        requireNonNull(claimAmount);
+        int claimAmountInt;
+        try {
+            claimAmountInt = Integer.parseInt(claimAmount.trim());
+            if (claimAmountInt < 0) {
+                throw new ParseException("Unplayable, Claim Amount cannot be negative");
```

Comment:
Did you mean not payable 😅

Reply:
I thought it was a new word that I did not know of 😆

Reply:
no i was trolling LOL, pls ignore